lifetimes/estimation.py

```diff
@@ -262,10 +277,15 @@ def conditional_expected_average_profit(self, frequency=None, monetary_value=Non
         Returns:
             the conditional expectation of the average profit per transaction
         """
-        m = self.data['monetary_value'] if monetary_value is None else monetary_value
-        x = self.data['frequency'] if frequency is None else frequency
+        if monetary_value is None:
+            monetary_value = self.data['monetary_value']
+        if frequency is None:
+            frequency = self.data['frequency']
         p, q, v = self._unload_params('p', 'q', 'v')
-        return (((q - 1) / (p * x + q - 1)) * (v * p / (q - 1))) + (p * x / (p * x + q - 1)) * m
+        # The expected average profit is a weighted average of individual monetary value and the population mean.
+        individual_weight = p * frequency / (p * frequency + q - 1)
+        population_mean = v * p / (q - 1)
```

Comment:
This is actually a bit confusing because the meaning of `v` in the paper is different. In the paper, `v` is called &gamma; (see equation 4 in http://www.brucehardie.com/papers/rfm_clv_2005-02-16.pdf), which is also a less-than-ideal choice given the many meanings of gamma...

In any case, I think that this change makes it easier to see how the prediction is made.